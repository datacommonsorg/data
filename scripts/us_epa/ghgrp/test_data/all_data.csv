dcid,sv,year,value
dcid:epaGhgrpFacilityId/1004377,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2010,0
dcid:epaGhgrpFacilityId/1004377,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2010,0
dcid:epaGhgrpFacilityId/1004377,dcid:Annual_Emissions_GreenhouseGas_MunicipalLandfills_NonBiogenic,2010,0
dcid:epaGhgrpFacilityId/1000112,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2010,82959.744
dcid:epaGhgrpFacilityId/1000112,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2010,82875.9
dcid:epaGhgrpFacilityId/1000112,dcid:Annual_Emissions_Methane_NonBiogenic,2010,38.25
dcid:epaGhgrpFacilityId/1000112,dcid:Annual_Emissions_NitrousOxide_NonBiogenic,2010,45.594
dcid:epaGhgrpFacilityId/1000112,dcid:Annual_Emissions_GreenhouseGas_StationaryCombustion_NonBiogenic,2010,362.8
dcid:epaGhgrpFacilityId/1000112,dcid:Annual_Emissions_GreenhouseGas_ElectricityGeneration_NonBiogenic,2010,82596.944
dcid:epaGhgrpFacilityId/1006394,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2010,25176.656
dcid:epaGhgrpFacilityId/1006394,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2010,25150.9
dcid:epaGhgrpFacilityId/1006394,dcid:Annual_Emissions_Methane_NonBiogenic,2010,11.75
dcid:epaGhgrpFacilityId/1006394,dcid:Annual_Emissions_NitrousOxide_NonBiogenic,2010,14.006
dcid:epaGhgrpFacilityId/1006394,dcid:Annual_Emissions_GreenhouseGas_StationaryCombustion_NonBiogenic,2010,25176.656
dcid:epaGhgrpFacilityId/1002885,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2010,38883.204
dcid:epaGhgrpFacilityId/1002885,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2010,38843.2
dcid:epaGhgrpFacilityId/1002885,dcid:Annual_Emissions_Methane_NonBiogenic,2010,18.25
dcid:epaGhgrpFacilityId/1002885,dcid:Annual_Emissions_NitrousOxide_NonBiogenic,2010,21.754
dcid:epaGhgrpFacilityId/1002885,dcid:Annual_Emissions_GreenhouseGas_StationaryCombustion_NonBiogenic,2010,38883.204
dcid:epaGhgrpFacilityId/1002707,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2010,63884.36
dcid:epaGhgrpFacilityId/1002707,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2010,63818.6
dcid:epaGhgrpFacilityId/1002707,dcid:Annual_Emissions_Methane_NonBiogenic,2010,30
dcid:epaGhgrpFacilityId/1002707,dcid:Annual_Emissions_NitrousOxide_NonBiogenic,2010,35.76
dcid:epaGhgrpFacilityId/1002707,dcid:Annual_Emissions_GreenhouseGas_StationaryCombustion_NonBiogenic,2010,63884.36
dcid:epaGhgrpFacilityId/1004377,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2011,194000
dcid:epaGhgrpFacilityId/1004377,dcid:Annual_Emissions_Methane_NonBiogenic,2011,194000
dcid:epaGhgrpFacilityId/1004377,dcid:Annual_Emissions_GreenhouseGas_MunicipalLandfills_NonBiogenic,2011,194000
dcid:epaGhgrpFacilityId/1010040,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2011,390393.5
dcid:epaGhgrpFacilityId/1010040,dcid:Annual_Emissions_Methane_NonBiogenic,2011,390393.5
dcid:epaGhgrpFacilityId/1010040,dcid:Annual_Emissions_GreenhouseGas_UndergroundCoalMines_NonBiogenic,2011,390393.5
dcid:epaGhgrpFacilityId/1010085,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2011,64664.5
dcid:epaGhgrpFacilityId/1010085,dcid:Annual_Emissions_Methane_NonBiogenic,2011,64664.5
dcid:epaGhgrpFacilityId/1010085,dcid:Annual_Emissions_GreenhouseGas_UndergroundCoalMines_NonBiogenic,2011,64664.5
dcid:epaGhgrpFacilityId/1000112,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2011,100591.828
dcid:epaGhgrpFacilityId/1000112,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2011,100489.9
dcid:epaGhgrpFacilityId/1000112,dcid:Annual_Emissions_Methane_NonBiogenic,2011,46.5
dcid:epaGhgrpFacilityId/1000112,dcid:Annual_Emissions_NitrousOxide_NonBiogenic,2011,55.428
dcid:epaGhgrpFacilityId/1000112,dcid:Annual_Emissions_GreenhouseGas_StationaryCombustion_NonBiogenic,2011,289.7
dcid:epaGhgrpFacilityId/1000112,dcid:Annual_Emissions_GreenhouseGas_ElectricityGeneration_NonBiogenic,2011,100302.128
dcid:epaGhgrpFacilityId/1006394,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2011,19389.57
dcid:epaGhgrpFacilityId/1006394,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2011,19368.4
dcid:epaGhgrpFacilityId/1006394,dcid:Annual_Emissions_Methane_NonBiogenic,2011,9.25
dcid:epaGhgrpFacilityId/1006394,dcid:Annual_Emissions_NitrousOxide_NonBiogenic,2011,11.92
dcid:epaGhgrpFacilityId/1006394,dcid:Annual_Emissions_GreenhouseGas_StationaryCombustion_NonBiogenic,2011,19389.57
dcid:epaGhgrpFacilityId/1003708,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2012,88826.52
dcid:epaGhgrpFacilityId/1003708,dcid:Annual_Emissions_SulfurHexafluoride_NonBiogenic,2012,88826.52
dcid:epaGhgrpFacilityId/1010640,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2012,7736.04
dcid:epaGhgrpFacilityId/1010640,dcid:Annual_Emissions_SulfurHexafluoride_NonBiogenic,2012,7736.04
dcid:epaGhgrpFacilityId/1010811,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2012,91298.04
dcid:epaGhgrpFacilityId/1010811,dcid:Annual_Emissions_SulfurHexafluoride_NonBiogenic,2012,91298.04
dcid:epaGhgrpFacilityId/1010812,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2012,14322.96
dcid:epaGhgrpFacilityId/1010812,dcid:Annual_Emissions_SulfurHexafluoride_NonBiogenic,2012,14322.96
dcid:epaGhgrpFacilityId/1009444,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2012,33518.28
dcid:epaGhgrpFacilityId/1009444,dcid:Annual_Emissions_SulfurHexafluoride_NonBiogenic,2012,33518.28
dcid:epaGhgrpFacilityId/1008026,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2013,75662.2
dcid:epaGhgrpFacilityId/1008026,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2013,1236.7
dcid:epaGhgrpFacilityId/1008026,dcid:Annual_Emissions_Methane_NonBiogenic,2013,74425.5
dcid:epaGhgrpFacilityId/1004034,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2013,13353.6
dcid:epaGhgrpFacilityId/1004034,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2013,16.1
dcid:epaGhgrpFacilityId/1004034,dcid:Annual_Emissions_Methane_NonBiogenic,2013,13337.5
dcid:epaGhgrpFacilityId/1004975,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2013,10821.8
dcid:epaGhgrpFacilityId/1004975,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2013,12.8
dcid:epaGhgrpFacilityId/1004975,dcid:Annual_Emissions_Methane_NonBiogenic,2013,10809
dcid:epaGhgrpFacilityId/1006481,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2013,3111.7
dcid:epaGhgrpFacilityId/1006481,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2013,3.7
dcid:epaGhgrpFacilityId/1006481,dcid:Annual_Emissions_Methane_NonBiogenic,2013,3108
dcid:epaGhgrpFacilityId/1007872,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2013,155513.8
dcid:epaGhgrpFacilityId/1007872,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2013,177.3
dcid:epaGhgrpFacilityId/1007872,dcid:Annual_Emissions_Methane_NonBiogenic,2013,155336.5
dcid:epaGhgrpFacilityId/1008026,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2014,75801.6
dcid:epaGhgrpFacilityId/1008026,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2014,1873.1
dcid:epaGhgrpFacilityId/1008026,dcid:Annual_Emissions_Methane_NonBiogenic,2014,73928.5
dcid:epaGhgrpFacilityId/1004034,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2014,13412.6
dcid:epaGhgrpFacilityId/1004034,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2014,16.1
dcid:epaGhgrpFacilityId/1004034,dcid:Annual_Emissions_Methane_NonBiogenic,2014,13396.5
dcid:epaGhgrpFacilityId/1004975,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2014,10870.5
dcid:epaGhgrpFacilityId/1004975,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2014,12.5
dcid:epaGhgrpFacilityId/1004975,dcid:Annual_Emissions_Methane_NonBiogenic,2014,10858
dcid:epaGhgrpFacilityId/1006481,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2014,2447.55
dcid:epaGhgrpFacilityId/1006481,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2014,2.8
dcid:epaGhgrpFacilityId/1006481,dcid:Annual_Emissions_Methane_NonBiogenic,2014,2444.75
dcid:epaGhgrpFacilityId/1007872,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2014,155671.85
dcid:epaGhgrpFacilityId/1007872,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2014,178.1
dcid:epaGhgrpFacilityId/1007872,dcid:Annual_Emissions_Methane_NonBiogenic,2014,155493.75
dcid:epaGhgrpFacilityId/1012507,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2016,44304.292
dcid:epaGhgrpFacilityId/1012507,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2016,16724.9
dcid:epaGhgrpFacilityId/1012507,dcid:Annual_Emissions_Methane_NonBiogenic,2016,27570.75
dcid:epaGhgrpFacilityId/1012507,dcid:Annual_Emissions_NitrousOxide_NonBiogenic,2016,8.642
dcid:epaGhgrpFacilityId/1012156,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2016,46921.114
dcid:epaGhgrpFacilityId/1012156,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2016,35166.1
dcid:epaGhgrpFacilityId/1012156,dcid:Annual_Emissions_Methane_NonBiogenic,2016,11734.75
dcid:epaGhgrpFacilityId/1012156,dcid:Annual_Emissions_NitrousOxide_NonBiogenic,2016,20.264
dcid:epaGhgrpFacilityId/1012155,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2016,78994.21
dcid:epaGhgrpFacilityId/1012155,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2016,65490.2
dcid:epaGhgrpFacilityId/1012155,dcid:Annual_Emissions_Methane_NonBiogenic,2016,13468.25
dcid:epaGhgrpFacilityId/1012155,dcid:Annual_Emissions_NitrousOxide_NonBiogenic,2016,35.76
dcid:epaGhgrpFacilityId/1012472,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2016,55733.472
dcid:epaGhgrpFacilityId/1012472,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2016,51896.2
dcid:epaGhgrpFacilityId/1012472,dcid:Annual_Emissions_Methane_NonBiogenic,2016,3810.75
dcid:epaGhgrpFacilityId/1012472,dcid:Annual_Emissions_NitrousOxide_NonBiogenic,2016,26.522
dcid:epaGhgrpFacilityId/1012466,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2016,177420.494
dcid:epaGhgrpFacilityId/1012466,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2016,57531.8
dcid:epaGhgrpFacilityId/1012466,dcid:Annual_Emissions_Methane_NonBiogenic,2016,119858
dcid:epaGhgrpFacilityId/1012466,dcid:Annual_Emissions_NitrousOxide_NonBiogenic,2016,30.694
dcid:epaGhgrpFacilityId/1000355,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2016,261382.84
dcid:epaGhgrpFacilityId/1000355,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2016,154176.7
dcid:epaGhgrpFacilityId/1000355,dcid:Annual_Emissions_Methane_NonBiogenic,2016,107115.25
dcid:epaGhgrpFacilityId/1000355,dcid:Annual_Emissions_NitrousOxide_NonBiogenic,2016,90.89
dcid:epaGhgrpFacilityId/1009263,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2016,3181.694
dcid:epaGhgrpFacilityId/1009263,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2016,1730.8
dcid:epaGhgrpFacilityId/1009263,dcid:Annual_Emissions_Methane_NonBiogenic,2016,1450
dcid:epaGhgrpFacilityId/1009263,dcid:Annual_Emissions_NitrousOxide_NonBiogenic,2016,0.894
dcid:epaGhgrpFacilityId/1009238,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2016,808767.966
dcid:epaGhgrpFacilityId/1009238,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2016,600130.6
dcid:epaGhgrpFacilityId/1009238,dcid:Annual_Emissions_Methane_NonBiogenic,2016,208416.25
dcid:epaGhgrpFacilityId/1009238,dcid:Annual_Emissions_NitrousOxide_NonBiogenic,2016,221.116
dcid:epaGhgrpFacilityId/1009283,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2016,57168.136
dcid:epaGhgrpFacilityId/1009283,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2016,12931.4
dcid:epaGhgrpFacilityId/1009283,dcid:Annual_Emissions_Methane_NonBiogenic,2016,44219.75
dcid:epaGhgrpFacilityId/1009283,dcid:Annual_Emissions_NitrousOxide_NonBiogenic,2016,16.986
