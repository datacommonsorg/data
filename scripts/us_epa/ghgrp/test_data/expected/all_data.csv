dcid,sv,year,value
dcid:epaGhgrpFacilityId/1000112,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2010,82959.744
dcid:epaGhgrpFacilityId/1000112,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2010,82875.9
dcid:epaGhgrpFacilityId/1000112,dcid:Annual_Emissions_Methane_NonBiogenic,2010,38.25
dcid:epaGhgrpFacilityId/1000112,dcid:Annual_Emissions_NitrousOxide_NonBiogenic,2010,45.594
dcid:epaGhgrpFacilityId/1000112,dcid:Annual_Emissions_GreenhouseGas_StationaryCombustion_NonBiogenic,2010,362.8
dcid:epaGhgrpFacilityId/1000112,dcid:Annual_Emissions_GreenhouseGas_ElectricityGeneration_NonBiogenic,2010,82596.944
dcid:epaGhgrpFacilityId/1006394,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2010,25176.656
dcid:epaGhgrpFacilityId/1006394,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2010,25150.9
dcid:epaGhgrpFacilityId/1006394,dcid:Annual_Emissions_Methane_NonBiogenic,2010,11.75
dcid:epaGhgrpFacilityId/1006394,dcid:Annual_Emissions_NitrousOxide_NonBiogenic,2010,14.006
dcid:epaGhgrpFacilityId/1006394,dcid:Annual_Emissions_GreenhouseGas_StationaryCombustion_NonBiogenic,2010,25176.656
dcid:epaGhgrpFacilityId/1011696,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2010,37277
dcid:epaGhgrpFacilityId/1011696,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2010,0
dcid:epaGhgrpFacilityId/1011696,dcid:Annual_Emissions_Methane_NonBiogenic,2010,37277
dcid:epaGhgrpFacilityId/1011696,dcid:Annual_Emissions_CarbonDioxide_Biogenic,2010,1.7
dcid:epaGhgrpFacilityId/1011696,dcid:Annual_Emissions_GreenhouseGas_StationaryCombustion_NonBiogenic,2010,0
dcid:epaGhgrpFacilityId/1011696,dcid:Annual_Emissions_GreenhouseGas_MunicipalLandfills_NonBiogenic,2010,37277
dcid:epaGhgrpFacilityId/1004063,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2010,17777.684
dcid:epaGhgrpFacilityId/1004063,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2010,17759.6
dcid:epaGhgrpFacilityId/1004063,dcid:Annual_Emissions_Methane_NonBiogenic,2010,8.25
dcid:epaGhgrpFacilityId/1004063,dcid:Annual_Emissions_NitrousOxide_NonBiogenic,2010,9.834
dcid:epaGhgrpFacilityId/1004063,dcid:Annual_Emissions_GreenhouseGas_StationaryCombustion_NonBiogenic,2010,17777.684
dcid:epaGhgrpFacilityId/1010527,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2012,0
dcid:epaGhgrpFacilityId/1009620,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2012,100753.2
dcid:epaGhgrpFacilityId/1009620,dcid:Annual_Emissions_SulfurHexafluoride_NonBiogenic,2012,100753.2
dcid:epaGhgrpFacilityId/1000355,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2013,163013.8
dcid:epaGhgrpFacilityId/1000355,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2013,59015.6
dcid:epaGhgrpFacilityId/1000355,dcid:Annual_Emissions_Methane_NonBiogenic,2013,103953.5
dcid:epaGhgrpFacilityId/1000355,dcid:Annual_Emissions_NitrousOxide_NonBiogenic,2013,44.7
dcid:epaGhgrpFacilityId/1008232,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2013,40908.05
dcid:epaGhgrpFacilityId/1008232,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2013,297.3
dcid:epaGhgrpFacilityId/1008232,dcid:Annual_Emissions_Methane_NonBiogenic,2013,40610.75
dcid:epaGhgrpFacilityId/1012491,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2016,219181.668
dcid:epaGhgrpFacilityId/1012491,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2016,174989.4
dcid:epaGhgrpFacilityId/1012491,dcid:Annual_Emissions_Methane_NonBiogenic,2016,44038.5
dcid:epaGhgrpFacilityId/1012491,dcid:Annual_Emissions_NitrousOxide_NonBiogenic,2016,153.768
dcid:epaGhgrpFacilityId/1012814,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2016,51315.808
dcid:epaGhgrpFacilityId/1012814,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2016,47713.4
dcid:epaGhgrpFacilityId/1012814,dcid:Annual_Emissions_Methane_NonBiogenic,2016,3469.5
dcid:epaGhgrpFacilityId/1012814,dcid:Annual_Emissions_NitrousOxide_NonBiogenic,2016,132.908
dcid:epaGhgrpFacilityId/1001277,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2017,110810.65
dcid:epaGhgrpFacilityId/1001277,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2017,133.4
dcid:epaGhgrpFacilityId/1001277,dcid:Annual_Emissions_Methane_NonBiogenic,2017,110677.25
dcid:epaGhgrpFacilityId/1001277,dcid:Annual_Emissions_NitrousOxide_NonBiogenic,2017,6392
dcid:epaGhgrpFacilityId/1011115,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2017,33870.3
dcid:epaGhgrpFacilityId/1011115,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2017,40.8
dcid:epaGhgrpFacilityId/1011115,dcid:Annual_Emissions_Methane_NonBiogenic,2017,33829.5
dcid:epaGhgrpFacilityId/1004377,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2019,518680.25
dcid:epaGhgrpFacilityId/1004377,dcid:Annual_Emissions_Methane_NonBiogenic,2019,518680.25
dcid:epaGhgrpFacilityId/1004377,dcid:Annual_Emissions_GreenhouseGas_MunicipalLandfills_NonBiogenic,2019,518680.25
dcid:epaGhgrpFacilityId/1007606,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2019,916250.412
dcid:epaGhgrpFacilityId/1007606,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2019,915046.8
dcid:epaGhgrpFacilityId/1007606,dcid:Annual_Emissions_Methane_NonBiogenic,2019,445.5
dcid:epaGhgrpFacilityId/1007606,dcid:Annual_Emissions_NitrousOxide_NonBiogenic,2019,758.112
dcid:epaGhgrpFacilityId/1007606,dcid:Annual_Emissions_GreenhouseGas_StationaryCombustion_NonBiogenic,2019,2104.992
dcid:epaGhgrpFacilityId/1007606,dcid:Annual_Emissions_GreenhouseGas_CementProduction_NonBiogenic,2019,914145.42
dcid:epaGhgrpFacilityId/1011696,dcid:Annual_Emissions_GreenhouseGas_NonBiogenic,2019,82134.75
dcid:epaGhgrpFacilityId/1011696,dcid:Annual_Emissions_CarbonDioxide_NonBiogenic,2019,0
dcid:epaGhgrpFacilityId/1011696,dcid:Annual_Emissions_Methane_NonBiogenic,2019,82134.75
dcid:epaGhgrpFacilityId/1011696,dcid:Annual_Emissions_GreenhouseGas_StationaryCombustion_NonBiogenic,2019,0
dcid:epaGhgrpFacilityId/1011696,dcid:Annual_Emissions_GreenhouseGas_MunicipalLandfills_NonBiogenic,2019,82134.75
